apps/graphql/lib/graphql/schema/repository.ex

```diff
@@ -218,6 +218,8 @@ defmodule GraphQl.Schema.Repository do
     field :auth_method,   non_null(:oidc_auth_method)
     field :configuration, :ouath_configuration, resolve: &GraphQl.Resolvers.OAuth.resolve_configuration/2
 
+    field :consent,  :consent_request
```

Comment:
@michaeljguarino This shouldn't be part of the OIDC provider, since it isn't related to the OIDC Provider setup. The `oauthConsent` query should return `Repository` as well as `consent_request`.

Reply:
it's only filled in the oauth consent request, which actually returns an instance of the oidc provider I believe

Reply:
The `oidcProvider` is part of the `installation` and i.e. when I was trying to log in to the console with a link that @DavidSpek shared with me, the `installation` object was empty for me. The structure seems to look like that.

```
oauthConsent {
      installation {
          oidcProvider {
              consent {
                  requestedScope
                  skip
              }
          }
      }
  }
```